src/test/java/de/muenchen/mobidam/s3/S3ArchiveTest.java

```diff
@@ -78,6 +78,9 @@ class S3ArchiveTest {
     @Value("${mobidam.archive.expiration-months:1}")
     private int expiration;
 
+    @SystemStub
```

Comment:
Dieses Stubbing brauchst Du nur, weil der EnvironmentReader keine Bean mehr ist, sondern im common als static definiert wurde. Dadurch schränkst Du die Testbarkeit und auch die Austauschbarkeit stark ein. Die ursprüngliche Definition sah so aus:

public class EnvironmentReader {
    private Environment environment;

    public String getEnvironmentVariable(final String key) {
        return this.environment.getProperty(key);
    }
}
Warum hast Du das in den commons geändert?

Reply:
Bist Du sicher, dass das in der CAP-Umgebung funktioniert?

Reply:
OK, aber wir brauchen den Reader trotzdem als Bean, damit man ihn leichter austauschen kann (z.B. im Test).

Reply:
Ist in Arbeit : https://github.com/it-at-m/mobidam-eai-commons-starter/pull/14